### DISCUSSION TITLE
미니 프로젝트 선정

### DISCUSSION THREAD

**Discussion by jgjgill** [prgrms-web-devcourse/React-Playground-Study#18] [Project]
지난 회고에서 나온 문제점..!
- 너무 자율적인 현 상황! 중구난방!
- 다같이 모일 수 있는 프로젝트의 필요성! 쉽지만 많은 것들을 연습할 수 있는 단기간 프로젝트!
- 이제는 쳐내야할 시간!

저는 2가지 정도가 좋은 것 같습니다..!
첫번째는 회고에서도 말했듯이 NextJS를 활용한 팀 블로그 만들기!
이거는 저도 하고 싶은 거여서 꼭 진행하고 싶네요!! 😚

회고에서 나온 것..!
간단한 숫자 퍼즐 게임 같은거 만들기
- 상태관리, 상호작용, 애니메이션 효과가 많이 필요해서 도움이 될 것 같다.

그 다음으로 생각한 것은 학습 공유 목적으로 진행하고 있는 docs를 더 예쁘게 꾸미고 싶은데요..!
이거는 [Docusaurus](https://docusaurus.io/)라는 기술을 사용해보고 싶다는 생각을 하게 되었습니다!
예전에 이 기술을 한 번 써봐야지 라는 생각을 가지고 있었거든요!!

어찌보면 팀 블로그와 똑같은데 현재 저의 창의력으로는 여기까지가 한계네요.. 😇

회고 때 이야기하지 못했던 생각들 편하게 말씀해주세요!
감사합니다..! 🙇‍♂️

**Comment by hyoribogo** [comment]
저도 NextJS 블로그 만드는 거에 참여하고 싶습니다! 다만 커밋은 많이 못하겠지만 .. 그래도 계속해서 모르는 것 물어보고 하면서 익히고 싶어요!

**Comment by jkea1** [comment]
생각을 좀 해봤는데 저는 아직 리액트와 타입스크립트 사용이 미숙해서 여기에 좀 더 집중하면서 모르는 내용을 질문하고 할 것 같습니다. 그래서 리액트, ts, 깃헙 이렇게 돌아가면서 공부한 내용을 공유 할 것 같습니다. 하지만 NextJS 블로그 만들기 프로젝트도 꾸준히 팔로우 하면서 리뷰도 남기면서 공부하겠습니다~ 여유가 있으면 커밋도 하겠습니다! 
그럼 한번 더 조사하겠습니다! 해당 되는 이모지 클릭해주시거나 댓글 달아 주세요! 여러 개 선택하셔도 괜찮습니다!

1. 본격적으로 NextJS 블로그 만들기에 참여하고 싶다! 😄 
2. React+TS 를 이용해서 퍼즐 게임 만들기(혹은 다른 주제로)에 참여하고 싶다! 🎉
3. 개인 공부에 집중하면서 공부한 내용을 공유하고 질문하고 리뷰를 받는식으로 진행하고 싶다. 🚀
4. 올라오는 PR에 대해서는 리뷰하고 9월 본 프로젝트와 관련된 이슈에 관해서는 의견을 나누겠지만 그 외 활동에는 참여하지 않겠다. 👀

다른 의견이 있으시면 자유롭게 이야기 해 주세요! 🥰
